.ci/job_matrix_debug.yaml

```diff
@@ -7,7 +7,6 @@ registry_auth: swx-storage
 
 kubernetes:
   cloud: swx-k8s
-  nodeSelector: 'beta.kubernetes.io/os=linux'
```

Comment:
lets add kubernetes subsection config so we can configure jnlp and nodeSelector per $arch and use it later in Matrix to extract?, smth like that:
```
arch_table:
  ```

Reply:
Added.